default.json

```diff
@@ -11,6 +11,7 @@
   "npm": {
     "commitMessageTopic": "{{prettyDepType}} {{depName}}"
   },
+  "ignoreDeps": ["node"],
```

Comment:
You can preserve updates for dockerfiles and ci if you only exclude engine updates like:

```json
"packageRules": [
    {
      "description": "Don't bump engines field in package.json",
      "matchPackageNames": ["node"],
      "matchManagers": ["npm"],
      "matchDepTypes": ["engines"],
      "rangeStrategy": "auto"
    }
  ]
```
[Source](https://github.com/renovatebot/renovate/discussions/13521#discussioncomment-1956672)

Reply:
TBH it is also desired for me to manually upgrade engines in CI. We intentionally sometimes pin older version